src/scripts/main.js

```diff
@@ -1,3 +1,358 @@
 'use strict';
 
-// write code here
+const tableBody = document.querySelector('tbody');
+const tableHead = document.querySelector('thead');
+const rows = [...tableBody.rows];
+
+// Sorting table
+
+let typeOfSort = 'DESC';
+let sortPrevTarget;
+
+function sortTable(index, array, type) {
+  const text = rows[0].cells[index].innerText;
+
+  const sortedRow = type === 'ASC'
+    ? array.sort((rowA, rowB) => {
```

Comment:
DRY - don't repeat yourselft
Move this check ( type === 'ASC' ) - to the sort callback

Reply:
also, split to the variables